### ISSUE TITLE
Seurat v3.0 FindConservedMarkers

### ISSUE THREAD

**Issue by harinipras** [satijalab/seurat#1169]
I am using Seurat v3.0. In my combined object I have WT and PD(say) in stim just like the tutorial. And I have 3 cell types say "CD4+" , "CD8+" and "B" cells. When I try to run FindConservedMarkers, I get this error.

```javascript
FindConservedMarkers(obj, ident.1 = "CD4+", grouping.var = "stim")

Error in SetIdent.Seurat(object = object, cells = colnames(x = object),  : 
  argument "value" is missing, with no default
```
Would you mind helping me out?

Thanks

**Comment by andrewwbutler** [comment]
Could you post the `traceback()`?

**Comment by harinipras** [comment]
```javascript
traceback()
3: SetIdent.Seurat(object = object, cells = colnames(x = object), 
       ident.use = paste(Idents(object = object), object.var[, 1], 
           sep = "_"))
2: SetIdent(object = object, cells = colnames(x = object), ident.use = paste(Idents(object = object), 
       object.var[, 1], sep = "_"))
1: FindConservedMarkers(obj1, grouping.var = "stim", ident.1 = "NK T-cells")
```

**Comment by vbutty** [comment]
Based on my limited take on the code, it appears that the SetIdent() call embedded in the new FindConservedMarkers() function does not conform to Seurat3.0's syntax (identity being passed as the ident.use argument instead of value)... Editing the function accordingly and running it as a script appears to fix the issue (upon loading metap).

**Comment by andrewwbutler** [comment]
Yes, that appears to the be the issue here. However, this was fixed as of f463e5f4c7847e394df88c6d9dea1206b8defca2. Reinstalling the latest Seurat v3 should resolve the issue.
